TexasHoldem.c

```diff
@@ -1,7 +1,323 @@
+	//first deal for participating players
+	printf("First Deal: \n");
+	for(i = 1; i <= playerNum; i++)
+	{
+		do
+		{
+			randomCard(i, 1);
+			correct = CheckRand(i,1);
+		}while(correct == 1);
+		switch (i)
+		{
+			case 1:
+			printf("\tP1: Face = %d Value = %d \n", player1cards[0][0], player1cards[0][1]);
+			break;
+			case 2:
+			printf("\tP2: Face = %d Value = %d \n", player2cards[0][0], player2cards[0][1]);
+			break;
+			case 3:
+			printf("\tP3: Face = %d Value = %d \n", player3cards[0][0], player3cards[0][1]);
+			break;
+			case 4:
+			printf("\tP4: Face = %d Value = %d \n", player4cards[0][0], player4cards[0][1]);
+			break;
+		}
+	}
+	
+	//deal for cpu's (no display to screen)
+	for(i = 4; i > playerNum; i--)
+	{
+		do
+		{
+			randomCard(i, 1);
+			correct = CheckRand(i,1);
+		}while(correct == 1);
+	}
+	
+	//second deal
+	printf("\nSecond Deal:\n");
+	for(i = 1; i <= playerNum; i++)
+	{
+		do
+		{
+			randomCard(i, 2);
+			correct = CheckRand(i,2);
+		}while(correct == 1);
+		switch (i)
+		{
+			case 1:
+			printf("\tP1: Face = %d Value = %d \n", player1cards[1][0], player1cards[1][1]);
+			break;
+			case 2:
+			printf("\tP2: Face = %d Value = %d \n", player2cards[1][0], player2cards[1][1]);
+			break;
+			case 3:
+			printf("\tP3: Face = %d Value = %d \n", player3cards[1][0], player3cards[1][1]);
+			break;
+			case 4:
+			printf("\tP4: Face = %d Value = %d \n", player4cards[1][0], player4cards[1][1]);
+			break;
+		}
+	}
+	
+	//second deal for cpu's
+	for(i = 4; i > playerNum; i--)
+	{
+		do
+		{
+			randomCard(i, 2);
+			correct = CheckRand(i,2);
+		}while(correct == 1);
+	}
+}
+
+//choose number of players and returns number needed for betting
+int playerChoice(void)
+{
+	int playerNum;
+	do
+	{
+		printf("How many players are playing the game? (1-4) ");
+		scanf("%d", &playerNum);
+	}while(playerNum > 4 && playerNum < 1);
+	printf("\n%d player game \n\n", playerNum);
+	return(playerNum);
+}
+
+//defines the bets for the players and cpu's if applicable
+//i am not too familiar with betting in this game so I will leave it to one of you guys to figure it out
+//i made this function for the bets, but I don't know how the blinds and different roles factor in
+void DefineBet(int playerNum)
+{
+	double userbet;
+	//Start with first player as dealer, players are 1, 2, 3, and 4, going clockwise
+	int dealer = 1;
+
+	//Define initial bets
+	switch(playerNum)
+	{
+		//one player 3 cpu
+		case 1:
+		printf("Player %d is the dealer, player %d is the small blind, and player %d is the big blind", dealer, (dealer+1)%4, (dealer+2)%4;
+		//cpus are players 1, 2, and 3
+		if ((dealer+3)%4 == 4)
+		{
+			printf("What is Player %d's bet? ", (dealer+3)%4;
+			scanf("%lf", playerbets[(dealer+3)%4]);
+		}
+		
+		break;
+		
+		//two players two cpu
+		//cpus are players 1 and 2
+		case 2:
+		printf("Player %d is the dealer, player %d is the small blind, and player %d is the big blind", dealer, (dealer+1)%4, (dealer+2)%4;
```

Comment:
put a \n at the end of the printf in all the statements.

Reply:
also the player (blank) displays as zero 1 or 2, so we need to add one to the number.